config-scalecube-configuration-service/src/test/java/io/scalecube/config/service/ApiKey.java

```diff
@@ -0,0 +1,57 @@
+package io.scalecube.config.service;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public class ApiKey {
+
+  private String APIKey;
+  private String[] Permissions;
+
+  public ApiKey() {}
+
+  public ApiKey(String apiKey, String[] permissions) {
+    this.APIKey = apiKey;
+    this.Permissions = permissions;
+  }
+
+  /** @return the apiKey */
+  public String getApiKey() {
+    return this.APIKey;
+  }
+
+  /** @return the permissions */
+  public String[] getPermissions() {
+    return this.Permissions;
+  }
+
+  /* (non-Javadoc)
+   * @see java.lang.Object#hashCode()
+   */
+  @Override
+  public int hashCode() {
+    final int prime = 31;
+    int result = 1;
+    result = prime * result + Arrays.hashCode(this.Permissions);
+    result = prime * result + Objects.hash(APIKey);
+    return result;
+  }
+
+  /* (non-Javadoc)
```

Comment:
really informative java docs :+1:

Reply:
this kind of javadocs is created automatically in order to make a link to the original one transparently.

Reply:
so maybe let's change in your idea generation of these redundant javadocs or just remove always it yourself.

Reply:
this is a simple POJO class to demonstrate in tests the abilities of this project.